pages/options/index.js

```diff
@@ -108,25 +96,15 @@ class OptionsPage extends Component {
   }
 
   toggleHours() {
-    this.setState({ showHours: !this.state.showHours })
-    if (this.state.showHours) {
-      this.setHours('open')
-      this.setState({ hours: 'open' })
-    } else {
-      this.setHours('all')
-      this.setState({ hours: 'all' })
-    }
+    this.setState({ showHours: !this.state.showHours }, () => {
```

Comment:
Let's avoid the async callback on business logic and just pull out the new state.

```
   toggleHours() {
     const showHours = !this.state.showHours
     const hours = showHours ? 'open' : 'all'
     this.setState({ showHours, hours })
   }
```